packages/plugin-rows/src/editor/index.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react'
+import {
+  EditorContext,
+  getDocument,
+  getPlugins,
+  PluginState,
+  StatefulPluginEditorProps
+} from '@edtr-io/core'
+import { rowsState } from '..'
+import { Row } from './row'
+
+export const RowsEditor = (
+  props: StatefulPluginEditorProps<typeof rowsState>
+) => {
+  const rows = props.state
+  const store = React.useContext(EditorContext)
+  return (
+    <React.Fragment>
+      {rows.items.map((row, index) => {
+        const doc = getDocument(store.state, row.id)
+        const plugins = getPlugins(store.state)
+        return (
+          //TODO: remove this wrapper if its not needed
```

Comment:
still open?

Reply:
We need it for `key` at least, didn't check for `position: relative` though...

Reply:
Key can also be used in `React.Fragment`. We need it for the style though, I suppose